index.html

```diff
@@ -235,14 +235,12 @@ <h2>Software projects</h2>
             as the map rendering engine.
           </p>
         </li>
-        <li><a href="https://github.com/Maps4HTML/geoserver">GeoServer MapML Community Module</a>
+        <li><a href="https://github.com/Maps4HTML/geoserver">GeoServer MapML Extension</a>
           <p>
-            A GeoServer module that enables MapML output, supporting images, 
+            A GeoServer extension that enables MapML output, supporting images, 
             features and tiles. There is basic 
-            <a href="https://docs.geoserver.org/latest/en/user/community/mapml/index.html">documentation</a> 
-            available, as well as 
-            <a href="https://youtu.be/14XvqNf4jTE?list=PLA07raojXKMuJdyBMs32G9ChY4JQxnlbn">instructions</a> 
```

Comment:
I deleted the link to the instructions on Youtube as they pertain to the Community Module.